cpp/velox/substrait/SubstraitToVeloxExpr.cc

```diff
@@ -243,6 +243,30 @@ core::TypedExprPtr SubstraitVeloxExprConverter::toExtractExpr(
   VELOX_FAIL("Constant is expected to be the first parameter in extract.");
 }
 
+core::TypedExprPtr SubstraitVeloxExprConverter::toLambdaExpr(
+    const ::substrait::Expression::ScalarFunction& substraitFunc,
+    const RowTypePtr& inputType) {
+  // arguments names and types
+  std::vector<std::string> argumentNames;
+  argumentNames.reserve(substraitFunc.arguments().size() - 1);
```

Comment:
Please ensure `substraitFunc.arguments().size() >= 1` to avoid negative reserve.